### PR TITLE
Move to ghactions

### PR DESCRIPTION
**Pull request by tjurtsch**

### CODE REVIEW

File: syn/symbiflow-yosys-plugins/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,6 @@
-{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
-
 package:
   name: symbiflow-yosys-plugins
-  version: {{ version }}
+  version: 0.0
```
Comment (PiotrZierhoffer) [inline syn/symbiflow-yosys-plugins/meta.yaml]: is this change needed?

File: sim/verilator/run_test.sh [hunk 2]
```diff
@@ -3,6 +3,7 @@
 set -e
 set -x
 
+export CONDA_BUILD_SYSROOT=/Users/runner/sdk/MacOSX10.9.sdk
```
Comment (PiotrZierhoffer) [inline sim/verilator/run_test.sh:6]: is this something we should have on a generic level?

File: sim/icarus/meta.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ package:
 
 source:
   git_url: https://github.com/steveicarus/iverilog.git
-  git_rev: master
+  git_rev: 540555fca318f142fbddaac7c93b9dc66c1febdb
```
Comment (PiotrZierhoffer) [inline sim/icarus/meta.yaml:9]: This is a temporary solution that we'll merge and then revert after the package is built

**Review by PiotrZierhoffer — CHANGES_REQUESTED** [review]

File: syn/symbiflow-yosys-plugins/meta.yaml (hunk 1 above)
Comment (tjurtsch) [reply syn/symbiflow-yosys-plugins/meta.yaml]: It's not. Reverted it.

File: sim/verilator/run_test.sh (hunk 2 above)
Comment (tjurtsch) [reply sim/verilator/run_test.sh]: This SDK is downloaded and unpacked in this custom location by download_sdk.sh script. Without pointing to it test won't pass: https://github.com/litex-hub/litex-conda-eda/runs/1662891928?check_suite_focus=true
Wrong SDK is used

Comment (tjurtsch) [reply sim/verilator/run_test.sh:6]: Piotr means generic level as e.g. workflow level. We can't have this env variable defined for whole workflow, because Conda overrides these variables and creates separate env for running tests.

File: .github/workflows/build-packages.yml [hunk 4]
```diff
@@ -0,0 +1,639 @@
+name: build-packages
+on: push
```
Comment (ajelinski) [inline .github/workflows/build-packages.yml:2]: Change to `on: [push, pull_request]`

**Comment by ajelinski** [comment]
LGTM
